lib/query/query_supporters.rb

```diff
@@ -231,6 +231,11 @@ def self.full_filter_expr(np_id, query)
     if query[:search].present?
       expr = expr.and_where(%Q(
         supporters.fts @@ websearch_to_tsquery('english', $search)
+        OR (
+          supporters.phone IS NOT NULL
+          AND supporters.phone != ''
+          AND supporters.phone_index = (regexp_replace($search, '\\D','', 'g'))
+        )
```

Comment:
I think it would be good to add tests for this, what do you think?

Reply:
That's a good idea!